source/fe/fe_rannacher_turek.cc

```diff
@@ -0,0 +1,119 @@
+#include <deal.II/fe/fe_rannacher_turek.h>
+#include <deal.II/base/quadrature_lib.h>
+#include <algorithm>
+
+#include <sstream>
+
+
+DEAL_II_NAMESPACE_OPEN
+
+
+template <int dim>
+FE_RannacherTurek<dim>::FE_RannacherTurek(const unsigned int degree,
+                                          const unsigned int n_face_support_points) :
+  FE_Poly<PolynomialsRannacherTurek<dim>, dim>(
+    PolynomialsRannacherTurek<dim>(),
+    FiniteElementData<dim>(this->get_dpo_vector(),
+                           1,
+                           2,
+                           FiniteElementData<dim>::L2),
```

Comment:
I forgot -- is the Rannacher-Turek element really completely discontinuous, or is it at least H_div conforming?